### ISSUE TITLE
Macro launches a different game

### ISSUE THREAD

**Issue by TheLichKingg** [BuilderDolphin/dolphSol-Macro#84]
Whenever I leave the macro on overnight or while I'm at school, it launches a game called "desouza's place".
I don't understand this.

**Comment by renzz00** [comment]
your private server link might be wrong. the format of the link is below.
**# = number**

### https://www.roblox.com/games/15532962292/Sols-RNG?privateServerLinkCode=################################

to get a link like this, use the share code you got from the private server configuration page, then enter it in your browser. When it tries to open Roblox, copy the link. If it looks like the link above, you can then copy it here in the settings tab of the macro.
![image](https://github.com/BuilderDolphin/dolphSol-Macro/assets/148696025/c1bde815-2778-42a2-a84c-85555be8bd12)

**Comment by MarkGamez1** [comment]
No, I'm having this problem too. I dont know why its happening, specifically in the newest verison. I decided to not use dolphsol macro for now while this is happening.
Webhook:
![image](https://github.com/BuilderDolphin/dolphSol-Macro/assets/84801342/32999ecf-12e1-484e-b2e9-0ced3e9c8c15)
